### ISSUE TITLE
GrowthCraft machines won't open interface

### ISSUE THREAD

**Issue by EarthwormO** [FTBTeam/FTB-Modpack-Issues#4032]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.1.0

### Describe the bug

No error seems to be generated, but none of the GrowthCraft machines will open an interface when right-clicked on.  Specifically tested Fruit Press, Brewing Barrel and Growthcraft Smoker.  

### Steps to reproduce

Build machine, place in world and right-click on it, nothing happens.  No err, no crash, just nothing happens.

### Expected behaviour

Interface should appear for these machines.

**Comment by Spaceferret42** [comment]
also broken is the fermenting barrel,  and roaster. In fact this whole mod seems broken

**Comment by DinnerBeef** [comment]
Try shift clicking them to open instead.

**Comment by Wickedsrain** [comment]
This is still not fixed.. you cannot interact with the press and the like from growthcraft. I know I try everyday. Shift clicking, clicking, nothing.  Modpack FTB Presents Direwolf20 1.20 1.4.0

**Comment by Spaceferret42** [comment]
I noticed that it is somewhat fixed by shift-clicking with an open hand on all the growth craft blocks also the book has been fixed

Sent from Yahoo Mail on Android 
 
  On Tue, Nov 7, 2023 at 9:41 AM, ***@***.***> wrote:   


This is still not fixed.. you cannot interact with the press and the like from growthcraft. I know I try everyday. Shift clicking, clicking, nothing. Modpack FTB Presents Direwolf20 1.20 1.4.0
